internal/command/action.go

```diff
@@ -200,3 +216,23 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+// printProviderVersions returns a string of all provider versions
+func printProviderVersions()(string, error){
+	var providerVersions bytes.Buffer
+	
+	providers, err := prov.GetAllProviders()
+	if err != nil {
+		return "", err
+	}
+
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(prov.DefaultPath, provider), "--version").Output()
+		if err != nil {
+			providerVersions.WriteString(fmt.Sprintf("%s version uknown\n", provider))
```

Comment:
Spelling is still busted here

Reply:
I commented on the other one that it's keeping it consistent with `summon --version`. Should we move away from that consistency? https://github.com/cyberark/summon/pull/138#discussion_r390513305

Reply:
Changed